src/lib/utils/operatorMetadataHelper.ts

```diff
@@ -172,3 +179,16 @@ export const isLink = (value: string) => {
     const linkRegex = /^(https?:\/\/)?[a-zA-Z0-9][\w.-]*\.[a-zA-Z]{2,}(\/\S*)?$/;
     return !linkRegex.test(value);
 };
+
+export const checkDkgAddress = (value: string) => {
```

Comment:
because we do it in another place. Check here https://github.com/bloxapp/ssv-web/blob/da/metadata-dkg-address/src/app/common/stores/applications/SsvWeb/OperatorMetadata.store.ts#L150